ecosystem/sep-0009.md

```diff
@@ -6,120 +6,151 @@ Title: Standard KYC Fields
+| -------------------------------------- | ------ | -------------------------------------------------------------------------------- |
+| `organization.name`                    | string | Full organization name as on the incorporation                                   |
+| `organization.VAT_number`              | string | Organization VAT number                                                          |
+| `organization.registration_number`     | string | Organization registration                                                        |
+| `organization.registration_date`       | string | Date the organization was registered                                             |
+| `organization.registered_address`      | string | Organization registered address                                                  |
+| `organization.number_of_shareholders`  | number | Organization shareholder number                                                  |
+| `organization.shareholder_name`        | string | Can be an organization or a person                                               |
+| `organization.photo_incorporation_doc` | binary | Image of incorporation documents                                                 |
+| `organization.photo_proof_address`     | binary | Image of a utility bill, bank statement with the organization's name and address |
+| `organization.address_country_code`    | string | country code for current address                                                 |
+| `organization.state_or_province`       | string | name of state/province/region/prefecture                                         |
+| `organization.city`                    | string | name of city/town                                                                |
+| `organization.postal_code`             | string | Postal or other code identifying organization's locale                           |
+| `organization.director_name`           | string | Organization registered managing director                                        |
+| `organization.website`                 | string | Organization website                                                             |
+| `organization.email`                   | string | Organization contact email                                                       |
+| `organization.phone`                   | string | Organization contact phone                                                       |
+
+## Addresses
+
+Address formatting varies widely from country to country and even within each country. See
+[here](https://stackoverflow.com/questions/11160192/how-to-parse-freeform-street-postal-address-out-of-text-and-into-components)
+for details. Rather than attempting to create a field for each possible part of an address in every country, this
+protocol takes a middle of the road approach. Address fields that are fairly universal can be encoded with the
```

Comment:
What if address is universal, but we also need to include street name + building number?
What do we place in `address? Is it only building address, or full address?

Reply:
I think building/apt/suite numbers are typically separate optional fields in address forms. Maybe we should also have a separate field so we don't have this ambiguity?

Reply:
This section hasn't been changed in this PR, so I suggest we keep things as-is for now.

Reply:
Yes, but do we have any recommendations/guidance for it?

Reply:
I don't think we do at the moment

Reply:
Got it. I think we should add some guidance for this field because right now it's quite confusing